vmm/src/config.rs

```diff
@@ -678,6 +691,64 @@ impl CpusConfig {
     }
 }
 
+impl PCISegmentConfig {
```

Comment:
nit: I think the rest of the code base has it capitalised like `Pci` - would you might adjusting to reflect that?

Reply:
Updated